common/namespace/registry/registry.go

```diff
@@ -42,23 +42,12 @@ import (
 	"go.temporal.io/server/common/log"
 	"go.temporal.io/server/common/log/tag"
 	"go.temporal.io/server/common/metrics"
+	"go.temporal.io/server/common/namespace"
 	"go.temporal.io/server/common/persistence"
 	"go.temporal.io/server/common/pingable"
 	"go.temporal.io/server/internal/goro"
 )
 
-// ReplicationPolicy is the namespace's replication policy,
-// derived from namespace's replication config
-type ReplicationPolicy int
```

Comment:
This was kept in the namespace package. Its only used there. Might make more sense to move it later 🤷